releng/org.eclipse.n4js.targetplatform/N4JS.setup

```diff
@@ -222,10 +222,10 @@
       xsi:type="setup.p2:P2Task"
       label="XSemantics">
     <requirement
-        name="it.xsemantics.sdk.feature.group"
-        versionRange="[1.12.0,1.12.1)"/>
+        name="org.eclipse.xsemantics.sdk.feature.group"
+        versionRange="[1.12.1,1.12.2)"/>
```

Comment:
Was the new version Lorenzo's decision?

Reply:
I guess yes. It is the only version published.
It works with our codebase (as far as our tests are concerned).